src/Microsoft.SqlTools.ServiceLayer/LanguageServices/PeekDefinition.cs

```diff
@@ -61,7 +62,7 @@ internal PeekDefinition(ServerConnection serverConnection, ConnectionInfo connIn
             Initialize();
         }
 
-        private Database Database
+        internal Database Database
```

Comment:
No because the peek definition object persists for only one peek definition call.